src/main/java/com/github/dockerjava/api/model/BindOptions.java

```diff
@@ -0,0 +1,44 @@
+package com.github.dockerjava.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.dockerjava.core.RemoteApiVersion;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+import java.io.Serializable;
+
+/**
+ * @since {@link RemoteApiVersion#VERSION_1_24}
+ */
+public class BindOptions implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty("Propagation")
+    BindPropagation propagation;
```

Comment:
That's why i said:
> and now we have half classes marked with since.

You can't trust to existing missing sinces and nobody will scan and write versions for old fields.